src/main/java/com/rednavis/vaadin/showcase/views/login/LoginView.java

```diff
@@ -34,14 +41,47 @@
   @Id("content")
   private Div content;
 
+  private TextField userName;
```

Comment:
I think we need move field declarations to template
https://vaadin.com/docs/v14/flow/polymer-templates/tutorial-template-intro.html

Something like this
```
    <div class="row">
      <vaadin-text-field id="field" class="field" placeholder="[[fieldPlaceholder]]" value="{{fieldValue}}" on-focus="_onFieldFocus" 
        on-blur="_onFieldBlur" theme="white">
        <iron-icon icon\$="[[fieldIcon]]" slot="prefix"></iron-icon>
      </vaadin-text-field>
      <vaadin-checkbox class="checkbox desktop" checked="{{checkboxChecked}}" on-focus="_onFieldFocus" 
        on-blur="_onFieldBlur">[[checkboxText]]</vaadin-checkbox>
      <vaadin-button id="clear" class="clear-btn" theme="tertiary">
        [[clearText]]
      </vaadin-button>
      <vaadin-button id="action" class="action-btn" theme="primary">
        <iron-icon icon\$="[[buttonIcon]]" slot="prefix"></iron-icon>
        [[buttonText]]
      </vaadin-button>
    </div>
```

You can download good examples here
https://vaadin.com/start/latest

Also we can separate template and styles
https://vaadin.com/docs/v14/flow/importing-dependencies/tutorial-include-css.html
https://vaadin.com/docs/v14/flow/theme/theming-overview.html

Reply:
Yes, I agree with a point of creating dom elements in js and I re-work PR.